### ISSUE TITLE
Add web support

### ISSUE THREAD

**Issue by doppio** [fayeed/flutter_freshchat#26]
It would be nice if this plugin supported the Freshdesk web SDK through the same API as it uses for iOS/Android. With Flutter web support in beta, it would be useful to make this plugin more cross-platform.

I may eventually implement this myself when I need it for my project, but that won't be for 6-12 months.

**Comment by chaitanya-y** [comment]
How to generate restoreId in flutter fresh chat(getting empty string)?

I checked some issues online.someone mentioned that restoreId will get generated only after first was sent.I tried to call send message function and then did identify user .Still, I am getting restoreId as empty string.

  FreshchatUser user = FreshchatUser.initail();
  user.email = email;
  user.firstName = firstName;
  user.lastName = lastName;
  user.phoneCountryCode = '91';
  user.phone = phone;
  bool result = await FlutterFreshchat.updateUserInfo(user: user);
  bool sendMessageSuccess =await FlutterFreshchat.send(message: 'Hello',tag: 'test');
  if(sendMessageSuccess){
  freshChatId = await FlutterFreshchat.identifyUser(externalID: phone, restoreID: null);
  print('freshChatId');
   }

and
2)Is there any way to trigger an event on the click of back button in the flutter chat screen.

**Comment by doppio** [comment]
@chaitanya-y You should file a new issue for that or ask on Stack overflow, this issue is about adding web support to the package.

**Comment by fayeed** [comment]
@doppio I haven't looked at the Freshchat web SDK or don't know much about it but I might integrate if enough people request it.

**Comment by kaumudpa** [comment]
It looks like WEB integration is needed.
